client/my-sites/plans/plan-overview/plan-status/index.jsx

```diff
@@ -31,6 +33,28 @@ const PlanStatus = React.createClass( {
 		page( `/checkout/${ this.props.selectedSite.slug }` );
 	},
 
+	renderNotice() {
+		const { plan } = this.props;
+
+		if ( isInGracePeriod( plan ) ) {
+			const daysAfterUserFacingExpiry = Math.abs( getDaysUntilUserFacingExpiry( plan ) ),
+				noticeText = daysAfterUserFacingExpiry === 0
+					? this.translate( 'Expired today' )
+					: this.translate(
+						'Expired %(daysAfterUserFacingExpiry)d day ago',
+						'Expired %(daysAfterUserFacingExpiry)d days ago', {
```

Comment:
We shouldn't probably use `daysAfterUserFacingExpiry` here since translators would not know what that is (I'm not sure either). What about just `daysAfterExpiry` or just `days` instead?